controller/src/main/java/io/pravega/controller/fault/ControllerClusterListener.java

```diff
@@ -57,7 +58,7 @@ public ControllerClusterListener(final Host host, final Cluster cluster,
         this.host = host;
         this.cluster = cluster;
         this.executor = executor;
-        this.sweepers = sweepers;
+        this.sweepers = Lists.newArrayList(sweepers);
```

Comment:
Did you mean to use `Collections.unmodifiableList` or perhaps Guava's `ImmutableList` here?